src/modules/formBuilder/components/itemEditor/FormEditorItemProperties.tsx

```diff
@@ -105,9 +106,36 @@ const FormEditorItemProperties: React.FC<FormEditorItemPropertiesProps> = ({
 
   // Monitor changes to the FormItem.component field
   const itemComponentValue = watch('component');
+  const repeatsValue = watch('repeats');
   const hiddenValue = watch('hidden');
+  const pickListOptionsValue = watch('pickListOptions');
+  const pickListReferenceValue = watch('pickListReference');
   const hasEnableWhen = !!watch('enableWhen.0');
 
+  // Whereas `itemComponentValue` above contains the user-provided component override (null if not specified),
+  // `realItemComponent` is the actual component for the item. It's used specifically for Choice/Open Choice types,
+  // which have their own rules for whether to show a dropdown or radio if the user hasn't specified anything.
+  // If needed, we could support other components here, but right now this will be undefined for non-Choice types.
+  const realItemComponent = useMemo(() => {
+    if (
+      itemTypeValue &&
+      [ItemType.Choice, ItemType.OpenChoice].includes(itemTypeValue)
+    ) {
+      return chooseSelectComponentType(
+        itemComponentValue,
+        repeatsValue,
+        pickListOptionsValue?.length,
+        !pickListReferenceValue
+      );
+    }
+  }, [
+    itemTypeValue,
+    itemComponentValue,
+    repeatsValue,
+    pickListOptionsValue,
+    pickListReferenceValue,
+  ]);
```

Comment:
I would probably move this into the `ManagePickListOptions` component, since it's specific to pick-lists and choice options.